snapcraft/plugins/go.py

```diff
@@ -39,6 +39,10 @@
       This entry tells the checked out `source` to live within a certain path
       within `GOPATH`.
       This is not needed and does not affect `go-packages`.
+
+    - go-buildtags:
+      (string)
```

Comment:
Should we support `go-installflags` instead? So people can use `-asmflags` or `-ldflags` in addition to `-tags`?